home/static/js/django_opportunities.js

```diff
@@ -0,0 +1,1154 @@
+    ],
+    "links": [
+      "https://djangonaut.space/",
+      "https://djangonaut.space/sessions/2025-session-4/",
+      "https://djangonaut.space/comms/2025-opening-session-4/"
+    ],
+    "description": "Djangonaut Space is a free, 8-week group mentoring program where individuals work self-paced in a semi-structured learning environment. This session will start on February 17th, 2025. Participants work in teams of 3-4 with mentors on Django core or third-party packages."
+  },
+  {
+    "name": "Djangonaut Space - Navigator (Mentor)",
+    "type": "mentorship",
+    "tags": ["mentorship", "djangonaut-space", "volunteer"],
+    "requirements": [
+      "Significant Django experience",
+      "Some Django contribution experience",
+      "Available for weekly mentoring sessions for 8 weeks",
+      "Patience and teaching ability"
+    ],
+    "outcomes": [
+      "mentorship skills",
+      "community engagement",
+      "supporting new contributors",
+      "leadership experience"
+    ],
+    "links": [
+      "https://djangonaut.space/",
+      "https://github.com/djangonaut-space/program"
+    ],
+    "description": "Navigators guide them on their contributing journey. A Navigator coaches a small group of 3-4 Djangonauts and attends weekly check-ins. Contact contact@djangonaut.space to express interest in mentoring."
+  },
+  {
+    "name": "Djangonaut Space - Captain",
+    "type": "mentorship",
+    "tags": ["mentorship", "djangonaut-space", "pastoral-care", "volunteer"],
+    "requirements": [
+      "Django community experience",
+      "Strong interpersonal skills",
+      "Available for one-on-one sessions",
+      "Empathy and supportiveness"
+    ],
+    "outcomes": [
+      "mentorship skills",
+      "community engagement",
+      "supporting new contributors",
+      "pastoral care experience"
+    ],
+    "links": [
+      "https://djangonaut.space/"
+    ],
+    "description": "Captains organize one to one sessions with Djangonauts to provide extra pastoral support and encouragement. They bring the warmth of the Django community to the program."
+  },
+  {
+    "name": "Djangonaut Space - Organizer/Admin",
+    "type": "administration",
+    "tags": ["mentorship", "djangonaut-space", "administration", "volunteer"],
+    "requirements": [
+      "Significant Django community experience",
+      "Strong organizational skills",
+      "Program management experience preferred",
+      "Commitment to diversity and inclusion"
+    ],
+    "outcomes": [
+      "leadership experience",
+      "program management skills",
+      "community building",
+      "organizational skills"
+    ],
+    "links": [
+      "https://djangonaut.space/",
+      "https://opencollective.com/djangonaut-space"
+    ],
+    "description": "Organizers coordinate the overall Djangonaut Space program, manage applications, match mentors with mentees, and ensure the program runs smoothly."
+  },
+  {
+    "name": "Django code contributions",
```

Comment:
Hi Tim. Great to see such a comprehensive list. One thing that would be nice to add to this section is testing pre-release versions and reporting bugs. Maybe that's two separate items, I don't know.

Reply:
That's a really good suggestion. I can definitely fit in the pre-release testing. I'll see what I can do about the reporting bugs. It's more of a "how do I phrase this" challenge.